src/python/pants/backend/shell/target_types.py

```diff
@@ -268,12 +269,77 @@ class ShellCommandOutputsField(StringSequenceField):
         Use a trailing slash on directory names, i.e. `my_dir/`.
         """
     )
+    removal_hint = "To fix, use `output_files` and `output_directories` instead."
+    removal_version = "2.17.0.dev0"
```

Comment:
Would it be possible to put this in a different PR, pretty please? And update the docs in that one as well, perhaps